example/test.re

```diff
@@ -1128,4 +1160,6 @@ let tests =
     <Section name="insertRule, the ultimate escape hatch">
       <div className="raw-css" />
     </Section>
+    <Section name="merging styles"> <button className=mergedStyles >
```

Comment:
there’s an extra space in the `<button>` tag 🙈

Reply:
My refmt is borked with the version of bs-platform included. Someone else might have to run it.